t/node/chash-hashon.t

```diff
@@ -625,3 +625,90 @@ chash_key: "chash_val_3"
 chash_key: "chash_val_4"
 chash_key: "chash_val_5"
 chash_key: "chash_val_6"
+
+=== TEST 13: set route(two upstream node, type chash), hash_on vars_combinations
```

Comment:
Style: need to run reindex:
https://github.com/apache/apisix/blob/9361dc8182053d224aeedb9c2edbf88365a95fc7/CONTRIBUTING.md#L123-L133

Reply:
get it